Jenkinsfile-subsquid

```diff
@@ -1,19 +1,20 @@
 @Library('jenkins-library')
 
 String agentLabel            = 'docker-build-agent'
-String envImageName          = 'node:16.18.1-bullseye-slim'
+String agentImage            = 'node:16-pnpm7'
 String subsquidToken         = 'subsquidToken'
 String targetProject         =  "fearless-x-moonriver"
 List   jobParams             = [
     string(defaultValue: targetProject, name: 'targetProject', trim: true),
     booleanParam(defaultValue: false, name: 'fullClean'),
     booleanParam(defaultValue: false, name: 'restart'),
     booleanParam(defaultValue: false, name: 'hardReset'),
+    booleanParam(defaultValue: false, name: 'deploy'),
     ]
 String registry               = 'docker.soramitsu.co.jp'
 String dockerBuildToolsUserId = 'bot-build-tools-ro'
 Boolean disableSecretScanner  = false
-String secretScannerExclusion = '.*squid.yaml'
+String secretScannerExclusion = '.*squid.yaml$|.docker-compose.yml\$'
```

Comment:
I think this can be solved in another way

Reply:
fixed. But anyway, docker-compose is empty and using only by developers for migrating the database before publishing.